UI/ModOptionsEntries.cs

```diff
@@ -681,7 +681,7 @@ public enum VariantCategory {
             if (includeRandomizer) {
                 menu.Add(new TextMenu.SubHeader(Dialog.Clean("MODOPTIONS_EXTENDEDVARIANTS_OSD")));
 
-                TextMenuOptionExt<int> text = getNonVariantScaleOption("OSD_TEXTOPACITY", "", new float[] { 0f, 0.1f, 0.2f, 0.3f, 0.4f, 0.5f, 0.6f, 0.7f, 0.8f, 0.9f, 1f },
```

Comment:
Removed the option to show variants without showing text, because it creates a loophole against the watermark purpose